tools/generator/CodeGeneratorContext.cs

```diff
@@ -16,5 +16,6 @@ public class CodeGeneratorContext
 		string ContextMethodString => ContextMethod != null ? "in method " + ContextMethod.Name + " " : null;
 		string ContextTypeString => ContextType != null ? "in managed type " + ContextType.FullName : null;
 		public string ContextString => ContextFieldString + ContextMethodString + ContextTypeString;
+		public string ContextTypeMember => (ContextType != null ? $"{ContextType.FullName}." : string.Empty) + ContextMethod?.Name ?? ContextField?.Name;
```

Comment:
Considering that this method performs a string allocation, should it perhaps instead be a `GetContextTypeMember()` *method*?